fenix/app/src/main/java/org/mozilla/fenix/home/recentbookmarks/view/RecentBookmarks.kt

```diff
@@ -180,23 +181,7 @@ private fun RecentBookmarkImage(bookmark: RecentBookmark) {
                     PlaceholderBookmarkImage()
                 }
 
-                WithIcon { icon ->
-                    Box(
-                        modifier = imageModifier.background(
-                            color = FirefoxTheme.colors.layer2,
-                        ),
-                        contentAlignment = Alignment.Center,
-                    ) {
-                        Image(
-                            painter = icon.painter,
-                            contentDescription = null,
-                            modifier = Modifier
-                                .size(36.dp)
-                                .clip(cardShape),
-                            contentScale = ContentScale.Crop,
-                        )
-                    }
-                }
+                FaviconFallbackBookmarkImage(bookmark)
```

Comment:
@Amejia481 is this what you had in mind? Just double-checking before landing

Reply:
Exactly!